psa-crypto/tests/aead.rs

```diff
@@ -26,15 +26,15 @@ const ENCRYPTED_DATA: [u8; 40] = [
 #[test]
 fn aead_encrypt_aes_ccm() {
     let alg = Aead::AeadWithDefaultLengthTag(AeadWithDefaultLengthTag::Ccm);
+    let mut usage_flags: UsageFlags = Default::default();
+    dbg!(usage_flags.set_encrypt(true));
```

Comment:
Could you please remove the `dbg!` macros (here and in other files)? So that other people that test locally don't have them printing something